topo/fot/forch/forch.yaml

```diff
@@ -20,6 +20,11 @@ orchestration:
     port_description: TAP
     sequester_timeout_sec: 360
     default_auto_sequestering: disabled
+    test_result_device_state:
```

Comment:
hard to tell from the code, but are there reasonable (functional) defaults for these? What happens if it's not defined or there's an unconfigured result?

Reply:
Defined here is what the default is in code. The default is the state machine dict on PortStateMachine, and anything defined here overwrites the state machine transitions.

Reply:
so what's the purpose of this bit then?  Is it actually testing anything?  I'm not sure how to get away with a simple integration test change for this...   or was this just meant as an example config to point people at?

Reply:
There’s another test that tests the opposite behavior in unit test.

Reply:
It is to test config ingestion, state machine state transition overwrites. The config still overwrites what is in the default even though they’re the same. By specifying the default and the result doesn’t change shows the introduction of the config doesn’t break anything. The unit test tests the other side; by specifying the opposite of default, the resulting device states changes.

Reply:
By this test alone, the config could be doing nothing and still not change results. With the unit tests and this test, it shows the config logic works as intended whether the config specifies nothing, or specifies devices go into infracted or specifies devices go into operational. Taking it out removes the testing of 1 of these scenarios.